### PR TITLE
[PKG-1484] Upgrade to 2.22.0

### PR DESCRIPTION
**Pull request by schuylermartin45**
- Upgrades to latest version of package
- Makes the linter happy

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,18 +1,19 @@
...
-  sha256: b4f5b48eaae1be83e2fd6fb77ac49d9b639be4ca4bd2e05b5368d29632a93725
+  sha256: 0f9f6752b0c534a7b22a6542b984fa6a2c18ab4d4e0a4c79f191138506a9a75f
 
 build:
   number: 0
-  skip: true  # [py<37]
-  script: {{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir --no-build-isolation -vvv
+  # jupyterlab_server isn't available on s390x yet.
+  skip: true  # [py<37 or s390x]
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation --ignore-installed --no-cache-dir -vvv
```
Comment (schuylermartin45) [inline recipe/meta.yaml:16]: I didn't re-order this for the sake or re-ordering. I didn't have the latest version locally and handled the conflict.

File: recipe/meta.yaml [hunk 2]
```diff
@@ -29,7 +30,6 @@ requirements:
     - json5 >=0.9.0
     - jsonschema >=4.17.3
     - jupyter_server >=1.21,<3
-    - packaging >=21.3
```
Comment (skupr-anaconda) [inline recipe/meta.yaml]: packaging >=21.3 is still required, see https://github.com/jupyterlab/jupyterlab_server/blob/v2.22.0/pyproject.toml#L36

**Review by akabanovs — APPROVED** [review]
Once addressed @skupr-anaconda 's comment, LGTM

File: recipe/meta.yaml (hunk 2 above)
Comment (schuylermartin45) [reply recipe/meta.yaml]: I'll suppress the linter then. Scary that it still builds without it.

**Review by skupr-anaconda — APPROVED** [review]

**Pull request merged**